discovery/kubernetes/kubernetes.go

```diff
@@ -91,6 +92,26 @@ type SDConfig struct {
 	Role               Role                         `yaml:"role"`
 	HTTPClientConfig   config_util.HTTPClientConfig `yaml:",inline"`
 	NamespaceDiscovery NamespaceDiscovery           `yaml:"namespaces,omitempty"`
+	Selectors          []ResourceSelectorConfigRaw  `yaml:"selectors,omitempty"`
```

Comment:
Maybe rename the structs as follows:
* `ResourceSelectorConfigRaw` -> `SelectorConfig`
* `roleSelectorConfig` -> `roleSelector`
* `resourceSelectorConfig` -> `resourceSelector`